cmd/root.go

```diff
@@ -84,6 +84,7 @@ var RootCmd = &cobra.Command{
 			"Description":     viper.GetString("description"),
 			"DiscoveryServer": viper.GetString("discovery-server"),
 			"AuthServers":     viper.GetStringMapString("auth-servers"),
+			"NOCServer":       viper.GetString("noc-server"),
```

Comment:
Can we have multiple Monitors as well?

This is the case when you have a simple monitor for a private network, but also want to report to the community NOC.